packages/app/src/app/pages/Dashboard/Components/Selection/DragPreview.tsx

```diff
@@ -145,7 +145,7 @@ export const DragPreview: React.FC<DragPreviewProps> = React.memo(
               position: 'absolute',
               border: viewMode === 'list' ? '1px solid' : 'none',
               borderColor: 'grays.700',
-              padding: viewMode === 'list' ? 2 : 0,
```

Comment:
Is it intended to remove the padding around the preview?

|Production|
|:-:|
|![Screen Shot 2023-02-01 at 22 44 57](https://user-images.githubusercontent.com/24959348/216210846-c27b5d5a-cdf3-4662-9c87-185a22709050.png)|

|Preview|
|:-:|
|![Screen Shot 2023-02-01 at 22 46 35](https://user-images.githubusercontent.com/24959348/216210914-5398f1a0-fb2b-4507-b927-c4fbcb396e84.png)|